src/Core/Binding/IBindingConfigProvider.cs

```diff
@@ -44,7 +45,7 @@ public interface IBindingConfigProvider
     /// For C++ it will be in a json file in a Sonar-specific format</remarks>
     public interface IBindingConfig
     {
-        string FilePath { get; }
+        IList<string> SolutionItems { get; }
```

Comment:
Does it need to be a list i.e. is the order important?

Reply:
The order is not important, no. I'll change to IEnumerable.